Framework/Geometry/test/ObjectTest.h

```diff
@@ -392,7 +393,7 @@ class ObjectTest : public CxxTest::TestSuite {
     checkTrackIntercept(track, expectedResults);
   }
 
-  void xtestTrackTwoIsolatedCubes()
+  void testTrackTwoIsolatedCubes()
```

Comment:
Oh yes, I did inadvertently reactivated that test. If it passes then I may as well leave it as it is.